### PR TITLE
Update navigation bar properties

### PR DESCRIPTION
**Pull request by yao-yu-neuroleap**

### CODE REVIEW

File: iOSTest/AppDelegate.m [hunk 1]
```diff
@@ -22,6 +23,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     self.navController = [[UINavigationController alloc] initWithRootViewController:mainMenuViewController];
     
+    UINavigationBar.appearance.barTintColor = UIColorFromRGB(0x0E5C89);
```
Comment (MiguelTepale) [inline iOSTest/AppDelegate.m]: ```Bartint``` looks good and I like the usage of the macro. For your learning experience, figure out how to change the ```navigation bar``` color in Objective-C. Google "change navigation bar tint color in objective-c" and let me know how that goes.

File: iOSTest/AppDelegate.m [hunk 2]
```diff
@@ -22,6 +23,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
+    UINavigationBar.appearance.tintColor = UIColor.whiteColor;
+    //UINavigationBar.appearance.titleTextAttributes =
```
Comment (MiguelTepale) [inline iOSTest/AppDelegate.m]: You need these two properties to change the text color. The first one is correct. You just need to finish the code that is commented out and the text should change to white.

**Review by MiguelTepale — COMMENTED** [review]
"Review changes" pressed by accident.

File: iOSTest/AppDelegate.m [hunk 3]
```diff
@@ -22,11 +22,19 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
     
     self.navController = [[UINavigationController alloc] initWithRootViewController:mainMenuViewController];
+<<<<<<< Updated upstream
     
     UINavigationBar.appearance.barTintColor = UIColorFromRGB(0x0E5C89);
     UINavigationBar.appearance.tintColor = UIColor.whiteColor;
     //UINavigationBar.appearance.titleTextAttributes =
+=======
+        
+>>>>>>> Stashed changes
```
Comment (MiguelTepale) [inline iOSTest/AppDelegate.m]: Including line 25: Please remove the version control markers. I cannot run the program.

File: iOSTest/AppDelegate.m [hunk 4]
```diff
@@ -22,11 +22,19 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     MenuViewController *mainMenuViewController = [[MenuViewController alloc] initWithNibName:@"MenuViewController" bundle:nil];
     
     self.navController = [[UINavigationController alloc] initWithRootViewController:mainMenuViewController];
+<<<<<<< Updated upstream
```
Comment (MiguelTepale) [inline iOSTest/AppDelegate.m]: Please remove version control markers.

File: iOSTest/AppDelegate.m [hunk 5]
```diff
@@ -22,11 +22,19 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
     
     UINavigationBar.appearance.barTintColor = UIColorFromRGB(0x0E5C89);
     UINavigationBar.appearance.tintColor = UIColor.whiteColor;
```
Comment (MiguelTepale) [inline iOSTest/AppDelegate.m]: Please remove any instances of Swift code in AppDelegate.m

File: iOSTest/AppDelegate.m [hunk 6]
```diff
@@ -22,11 +22,19 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
     //UINavigationBar.appearance.titleTextAttributes =
+=======
+        
+>>>>>>> Stashed changes
     // Use Objective-C if you plan to code in the app delegate
+    [[UINavigationBar appearance] setBarTintColor:UIColorFromRGB(0x0E5C89)];
```
Comment (MiguelTepale) [inline iOSTest/AppDelegate.m]: Please use ```UIColor``` class to ```setBarTintColor```.

Comment (yao-yu-neuroleap) [reply iOSTest/AppDelegate.m]: For me, I think this is the easiest way to use rgb color from what I am searching on stackoverflow. Can I just keep it this way?

Comment (MiguelTepale) [reply iOSTest/AppDelegate.m]: You know I won't be letting you off the hook that easily.

I understand that your solution was obtained quickly which is commendable but it doesn't adhere to the project requirements. Interestingly enough, your ```#define``` has the code to instantiate a ```UIColor``` object. I'll point you to this [article](https://stackoverflow.com/a/35626097/8865491) that will show you the way to do it. Once you have created the object and set it to ```setBarTintColor```. Please remove ```#define```.

Please do reach out if you're stuck or have questions. I'm always happy to help.

Comment (yao-yu-neuroleap) [reply iOSTest/AppDelegate.m]: Ok I will look into it. Thanks Miguel!

Comment (MiguelTepale) [reply iOSTest/AppDelegate.m]: Good job!

**Pull request merged**